### DISCUSSION TITLE
Tests for bug correction

### DISCUSSION THREAD

**Discussion by dbenielli** [archimede-institut/GINsim#2] [Announcements]
[bugs_improvements.pdf](https://github.com/user-attachments/files/17519641/bugs_improvements.pdf)

**Comment by dbenielli** [comment]
- Link to Issue
[https://github.com/archimede-institut/GINsim/issues/3](https://github.com/archimede-institut/GINsim/issues/3)

**Reply by chaouiya** [reply]
Working well!
